tutor4points/core/static/css/base.css

```diff
@@ -1,30 +1,27 @@
 .header h1 {
   text-align: center;
-  margin: 35px;
+  margin: 2.188rem;
   font-family: "Oswald", serif;
   font-size: 4rem;
   font-weight: bolder;
-  margin-top:70px;
+  margin-top:4.375rem;
```

Comment:
It might be useful to group similar characteristics together. For example, all the font together, all the margin values together, etc.